__tests__/app.test.js

```diff
@@ -36,11 +37,48 @@ describe('GET /api', () => {
         request(app)
         .get('/api')
         .expect(200)
-            .then(({body}) => {
-                expect(Object.keys(body.endpoints)).toHaveLength(3)
-                expect(Object.keys(body.endpoints).includes('GET /api')).toBe(true)
-                expect(Object.keys(body.endpoints).includes('GET /api/topics')).toBe(true)
-                expect(Object.keys(body.endpoints).includes('GET /api/articles')).toBe(true)
+        .then(({body}) => {
+            expect(Object.keys(body.endpoints)).toHaveLength(3)
+            expect(Object.keys(body.endpoints).includes('GET /api')).toBe(true)
+            expect(Object.keys(body.endpoints).includes('GET /api/topics')).toBe(true)
+            expect(Object.keys(body.endpoints).includes('GET /api/articles')).toBe(true)
+        })
+    })
+})
+
+describe('GET /api/articles/:article_id', () => {
+    it('returns status 200 and the article requested', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.article_request).toHaveLength(1)
+            expect(Object.keys(body.article_request[0]).includes('author')).toBe(true)
+            expect(Object.keys(body.article_request[0]).includes('title')).toBe(true)
+            expect(Object.keys(body.article_request[0]).includes('article_id')).toBe(true)
+            expect(Object.keys(body.article_request[0]).includes('body')).toBe(true)
+            expect(Object.keys(body.article_request[0]).includes('topic')).toBe(true)
+            expect(Object.keys(body.article_request[0]).includes('created_at')).toBe(true)
+            expect(Object.keys(body.article_request[0]).includes('votes')).toBe(true)
+            expect(Object.keys(body.article_request[0]).includes('article_img_url')).toBe(true)
```

Comment:
Can we think of a better way to test the article we are getting back?
Currently we could be getting any article back and not the one we requested.
Take a look at the Jest matchers, ObjectContaining and ToMatchObject.